lib/cn/kblock_builder.c

```diff
@@ -842,7 +852,7 @@ kbb_estimate_alen(struct cn *cn, size_t wlen, enum hse_mclass mclass)
  * This function unconditionally invokes kblock_free().
  */
 static merr_t
-kblock_finish(struct kblock_builder *bld, struct wbb *ptree)
+kblock_finish(struct kblock_builder *bld, struct wbb *ptree, struct hlog *hlog)
```

Comment:
AFAICT, all callers just seems to use bld->hlog, so why the change?

Reply:
My guess: because the last kblock will use a different hlog.

Reply:
Yea this was my mistake. Should be fixed in latest revision.